watertap/property_models/anaerobic_digestion/adm1_reactions.py

```diff
@@ -1551,35 +1540,27 @@ def rule_I_pH_aa(self):
         )
 
         def rule_I_pH_ac(self):
-            return pyo.Expr_if(
-                self.pH > self.params.pH_UL_ac,
-                1,
-                pyo.exp(
-                    -3
-                    * (
-                        (self.pH - self.params.pH_UL_ac)
-                        / (self.params.pH_UL_ac - self.params.pH_LL_ac)
-                    )
-                    ** 2
-                ),
+            return (
+                -3
+                * (
+                    smooth_max(0, self.params.pH_UL_ac - self.pH, eps=1e-8)
+                    / (self.params.pH_UL_ac - self.params.pH_LL_ac)
+                )
+                ** 2
             )
 
         self.I_pH_ac = pyo.Expression(
             rule=rule_I_pH_ac, doc="pH inhibition of acetate-utilizing microorganisms"
         )
 
         def rule_I_pH_h2(self):
-            return pyo.Expr_if(
-                self.pH > self.params.pH_UL_h2,
-                1,
-                pyo.exp(
-                    -3
-                    * (
-                        (self.pH - self.params.pH_UL_h2)
-                        / (self.params.pH_UL_h2 - self.params.pH_LL_h2)
-                    )
-                    ** 2
-                ),
+            return (
+                -3
+                * (
+                    smooth_max(0, self.params.pH_UL_h2 - self.pH, eps=1e-8)
+                    / (self.params.pH_UL_h2 - self.params.pH_LL_h2)
+                )
+                ** 2
```

Comment:
Technically, these are log(I_pH_ac) and log(I_pH_h2). We should name the associated expressions accordingly.